Sources/StreamChat/ChatClient+Environment.swift

```diff
@@ -127,7 +127,8 @@ extension ChatClient {
             _ offlineRequestsRepository: OfflineRequestsRepository,
             _ eventNotificationCenter: EventNotificationCenter,
             _ database: DatabaseContainer,
-            _ apiClient: APIClient
+            _ apiClient: APIClient,
+            _ channelListUpater: ChannelListUpdater
```

Comment:
Injecting `ChannelListUpdater` to `SyncRepository` which allows refetching channels for the query. Old way is leveraging ChannelListController directly.